tests/metrics/grant_availability.py

```diff
@@ -115,11 +114,10 @@ def metric_grant(alice, ursulas: Optional[Set[Ursula]] = None) -> Policy:
     policy_end_datetime = maya.now() + DURATION
     policy = alice.grant(threshold=THRESHOLD,
                          shares=SHARES,
-                         ursulas=handpicked_ursulas,
+                         # ursulas=handpicked_ursulas,
```

Comment:
Seems like the `ursulas` parameter is determined here - perhaps just update that code to be an empty set? That way this doesn't need to be commented out (although `handpicked_ursulas` still needs to be renamed to `ursulas`)

Reply:
relocated this commit to https://github.com/nucypher/nucypher/pull/2915